Results 5/N (Total 6/N)

jharenza opened the pull request:
<!--Hi there, thanks for your contribution! Please take a moment to fill out this template to facilitate the review of your pull request.-->

### Purpose

Transcriptomic overview figure 4 results section - UMAP clustering, GSVA, and Telomerase activity

<!--Check to make sure your title reflects the purpose and nature of your changes-->

### Issue

NA

### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.

#### Which areas should receive a particularly close look?
NA, whole section

#### Is there anything that you want to discuss further?
No

#### Is the pull request ready for review?
yes

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [ ] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### Spellcheck Step

The dictionary used for spellchecking can be updated.
Edit the file in `build/assets/custom-dictionary.txt` by adding new entries to the end.
You do not need to change anything else.
However, if you want to update the first line to have an accurate count of words and you want to remove non-unique ones, run the following command from within `build/assets` on your favorite OS X or Linux machine:
```
(( len = $(awk '!a[$0]++' < custom-dictionary.txt | wc -l ) - 1 )); tmpfile="$(mktemp)"; echo "personal_ws-1.1 en $len utf-8" > $tmpfile; tail -n +2 custom-dictionary.txt | awk '!a[$0]++' >> $tmpfile; mv $tmpfile custom-dictionary.txt
```

I will update the dictionary once #167 is in

jashapiro commented on content/04.results.md:
```diff
@@ -157,46 +157,22 @@ The number of SV and CNV breakpoints were significantly correlated across tumors
+#### Telomerase activity 
+
+We next used gene expression data to predict telomerase activity using EXpression-based Telomerase ENzymatic activity Detection (EXTEND) [@doi:10.1038/s41467-020-20474-9] as a surrogate measure of malignant potential [@doi:10.1038/s41467-020-20474-9; 10.1093/carcin/bgp268].
+EXTEND scores calculated using FPKM or counts were tightly correlated (p < 2.2e-16) for each library type (**Figure {@fig:FigS4}E-F**) and significantly correlated with _TERT_ and _TERC_ expression (**Figure {@fig:FigS4}G-J**).  
```
I feel like this sentence, while a good check, may not need to be in the main body. In some sense this is the expected recapitulation of the EXTEND paper?

jharenza replied:
yeah, I think this was written before the telomerase paper came out, so I tried to keep some of the original text, but you are right - I will shorten.

jashapiro commented on content/07.supplemental.md:
```diff
@@ -5,3 +5,5 @@
 
 ![Figure S3. Genomic instability of pediatric brain tumors, Related to Figures 2 and 3.](images/supplement/Figure-S3.png){#fig:FigS3}
+
+![Figure S4. Related to Figure 4. A, Pearson correlation for counts with FPKM from poly-A samples (R = 0.94, p < 2.2e-16, N = ). B, Pearson correlation for counts with FPKM from stranded samples (R = 0.97, p < 2.2e-16, N = ).](images/supplement/SuppTelomerase_Activities.png){#fig:FigS4}
```
This caption does not reflect the updated content of Fig S4.

jharenza replied:
updated this and will upload a new figure

jashapiro reviewed (commented):
This looks good. My largest comments are about the interpretation of UMAP plots, where I think it is important to be very circumspect, so I tried to mute some of the stronger conclusions.  

The other big thing I noted is that the caption for figure S4 is incomplete/inaccurate, so we should at least flag that as needing an update.

jharenza commented on content/04.results.md:
```diff
@@ -157,46 +157,22 @@ The number of SV and CNV breakpoints were significantly correlated across tumors
+
+We performed UMAP unsupervised dimension reduction to validate that brain tumors cluster by histology (**Figure {@fig:Fig4}A**) and geneset variant analysis (GSVA) for Hallmark cancer gene sets to demonstrate activation of underlying oncogenic pathways (**Figure {@fig:Fig4}B**).	
+We observed medulloblastoma samples cluster by molecular subtype with Groups 3 and 4 tightly clustered and WNT and SHH in distinct clusters (**Figure {@fig:FigS4}A**), as expected.
+Additionally, _C11orf95::RELA_ fusion positive ependymomas and _BRAF_-driven low-grade tumors formed distinct clusters.
```
yeah you are right, and we are not going to label those since we are going to color by subtype in the supplement, so i will update this and the figure

jharenza commented:
@jashapiro - I think I addressed your comments. In addition, Figures 4 and 5 are a bit in limbo right now. @pmadsen101 is going to work on the immune deconvolution pieces, so I am going to leave Figure 4C/D for now and stop here so it doesn't become too long. In addition, I think Figure 5A-G are good (next PR will add this info), but H is still in the works (not sure how much I like _all_ of those cor plots, so open to suggestions.

jashapiro commented on content/04.results.md:
```diff
@@ -161,46 +161,25 @@ The number of SV and CNV breakpoints were significantly correlated across tumors
+Additionally, _C11orf95::RELA_ fusion positive ependymomas formed distinct clusters (**Figure {@fig:FigS4}B**).
+However, within low-grade (**Figure {@fig:FigS4}C** and high-grade (**Figure {@fig:FigS4}D** astrocytic tumors, tumors were dispersed by molecular subtype across their respective clusters.
+
+We performed geneset variant analysis (GSVA) for Hallmark cancer gene sets, demonstrating activation of underlying oncogenic pathways (**Figure {@fig:Fig4}B**).	
```
```suggestion
We performed gene set variant analysis (GSVA) for Hallmark cancer gene sets, demonstrating activation of underlying oncogenic pathways (**Figure {@fig:Fig4}B**).	
```

jashapiro commented on build/assets/custom-dictionary.txt:
```diff
@@ -138,6 +141,7 @@ GATK
 GBM
 GENCODE
 gencode
+geneset
```
```suggestion
```

jashapiro reviewed (approved):
This looks good. I just had a minor spelling change. 

I thought I might save my fig 5 comments for the other PR, but I agree that showing all of the correlations seems unnecessary. Is there a reason to make them a main display figure rather than supplement?

jharenza commented:
> Is there a reason to make them a main display figure rather than supplement?

It's interesting - I'm not exactly sure the best way to do this but looking at broad histology vs cancer group vs subtypes reveals different patterns. I thought I might keep it broad and write about the histologies with the correlations (though I guess I can also pull out the significant ones for main?). I had colored them in the updated figure 5, which makes them less "blah", but also not entirely necessary for the main. The one I had in the supp is for subtypes, to highlight a few there...still thinking about the best way to do this.
